hw/ip/pwrmgr/doc/_index.md

```diff
@@ -327,12 +327,15 @@ Assume first the system has the power states described [above](#supported-low-po
 
 ### Possible Exits
 
-Once low power is entered, the system may exit due to several reasons.
+Once low power is initiated, the system may exit due to several reasons.
 1. Graceful low power exit - This exit occurs when some source in the system gracefully wakes up the power manager.
 2. System reset request - This exit occurs when either software or a peripheral requests the pwrmgr to reset the system.
 2. [Fall through exit](#fall-through-handling) - This exit occurs when an interrupt manages to break the wait-for-interrupt loop.
```

Comment:
Nit, but the enumeration here seems to be off...

Reply:
fixed!